plugins/hydra_rq_launcher/setup.py

```diff
@@ -6,7 +6,7 @@
     LONG_DESC = fh.read()
     setup(
         name="hydra-rq-launcher",
-        version="1.0.0rc1",
+        version="1.0.0rc2",
```

Comment:
As a rule, I try to always have the repo in the next version. (in other words, it should have already been at rc2).